src/sentry/static/sentry/app/views/organizationDiscover/intro.jsx

```diff
@@ -94,6 +94,8 @@ export default class Intro extends React.Component {
 const IntroContainer = styled(Flex)`
   font-size: ${p => p.theme.fontSizeMedium};
   color: ${p => p.theme.gray5};
+  min-width: 450px;
```

Comment:
Could we get these widths to inherit from BodyContent instead of defining on each child component? Also the parent container has a min-width of 325px which is smaller than this - can we keep these numbers consistent?

Reply:
The 325px is from the topbar container, which isn't its parent, unfortunately. I can change it to 450px to make it more consistent; 325 was just the minimum it could go before it would mess up the formatting.